error-handler.js

```diff
@@ -258,9 +258,17 @@ createHandler = function createHandler(options) {
         }
       },
 
+      shouldContinue = function
```

Comment:
> I want to be sure you see the usage of `status()`.

I believe that use is out of scope for this module. Users should be able to run their own custom check to see if there's a planned maintenance activity that warrants special handling. Otherwise we're holding state for the users just for the sake of holding state.

A better plan is to be a stateless as possible and let users keep track of their own application state.